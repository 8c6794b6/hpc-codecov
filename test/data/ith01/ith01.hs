{-# LANGUAGE TemplateHaskell #-}

module Main where

import TH01

$fooDecl

$barDecl

main :: IO ()
main = do
  print foo
  print bar

