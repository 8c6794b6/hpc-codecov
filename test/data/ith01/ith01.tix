Tix [ TixModule "Main" 3976838569 10 [1,1,1,1,1,1,1,1,1,1], TixModule "TH01" 899367073 4 [0,0,0,0]]
