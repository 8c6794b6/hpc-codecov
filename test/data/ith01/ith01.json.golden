{"coverage":{"test/data/ith01/ith01.hs":{"12":1,"13":1,"14":1},"test/data/ith01/./TH01.hs":{"8":0,"11":0}}}
